prometheus-metrics-exposition-formats/src/main/java/io/prometheus/metrics/expositionformats/OpenMetricsTextFormatWriter.java

```diff
@@ -241,33 +243,23 @@ private void writeUnknown(OutputStreamWriter writer, UnknownSnapshot snapshot) t
             writeNameAndLabels(writer, metadata.getPrometheusName(), null, data.getLabels());
             writeDouble(writer, data.getValue());
             if (exemplarsOnAllMetricTypesEnabled) {
-            writeScrapeTimestampAndExemplar(writer, data, data.getExemplar());
+                writeScrapeTimestampAndExemplar(writer, data, data.getExemplar());
             } else {
                 writeScrapeTimestampAndExemplar(writer, data, null);
             }
         }
     }
 
     private void writeCountAndSum(OutputStreamWriter writer, MetricMetadata metadata, DistributionDataPointSnapshot data, String countSuffix, String sumSuffix, Exemplars exemplars) throws IOException {
-        int exemplarIndex = 0;
         if (data.hasCount()) {
             writeNameAndLabels(writer, metadata.getPrometheusName(), countSuffix, data.getLabels());
             writeLong(writer, data.getCount());
-            if (exemplars.size() > 0) {
-                writeScrapeTimestampAndExemplar(writer, data, exemplars.get(exemplarIndex));
-                exemplarIndex = exemplarIndex + 1 % exemplars.size();
-            } else {
-                writeScrapeTimestampAndExemplar(writer, data, null);
-            }
+            writeScrapeTimestampAndExemplar(writer, data, exemplars.getLatest());
         }
         if (data.hasSum()) {
             writeNameAndLabels(writer, metadata.getPrometheusName(), sumSuffix, data.getLabels());
             writeDouble(writer, data.getSum());
-            if (exemplars.size() > 0) {
-                writeScrapeTimestampAndExemplar(writer, data, exemplars.get(exemplarIndex));
-            } else {
-                writeScrapeTimestampAndExemplar(writer, data, null);
-            }
+            writeScrapeTimestampAndExemplar(writer, data, exemplars.getLatest());
```

Comment:
What is the use case of having an exemplar on `_sum`? I'm thinking it might be unnecessary if there is no use-case for it.

Reply:
You're right, I removed Exemplars on `_sum`. Both for histograms and summaries.